src/coreclr/debug/createdump/createdumpwindows.cpp

```diff
@@ -8,7 +8,7 @@
 // The Windows create dump code
 //
 bool
-CreateDump(const char* dumpPathTemplate, int pid, const char* dumpType, MINIDUMP_TYPE minidumpType)
+CreateDump(const char* dumpPathTemplate, int pid, const char* dumpType, MINIDUMP_TYPE minidumpType, bool crashReport, int crashThread, int signal)
```

Comment:
On Windows, none of those new arguments will ever be set because of the #ifdef HOST_UNIX of the command line options in main.cpp. I don't think we really need to worry about this.